README.md

```diff
@@ -32,6 +32,12 @@ go run mage.go build
 
 You will find the WASM plugin under `./build/main.wasm`.
 
+### Multiphase
+
+By default, coraza-proxy-wasm runs with multiphase evaluation enabled (See [coraza.rule.multiphase_evaluation](.magefiles/magefile.go) build tag). It enables the evaluation of rule variables in the phases that they are ready for, potentially anticipating the phase the rule is defined for. This feature suits coraza-proxy-wasm, and specifically Envoy request lifecycle, aiming to inspect data that has been received so far as soon as possible. It leads to enforce actions the earliest possible, avoiding WAF bypasses. This functionality, in conjunction with the [early blocking CRS feature](#recommendations-using-crs-with-proxy-wasm), permits to effectively raise the anomaly score and eventually drop the request at the earliest possible phase.
```

Comment:
At some point we need to write a document in `coraza` explaining multiphase evaluation.

Reply:
Yes, definitely. I didn't go further here because the right place has to be the Coraza repo.